src/Parser/Core/HIT_TYPES.js

```diff
@@ -2,6 +2,10 @@ const HIT_TYPES = {
   NORMAL: 1,
   CRIT: 2,
   NOCLUEWHATTHISIS: 3, // seen at Aura of Sacrifice
+  BLOCKED_NORMAL: 4,
+  BLOCKED_CRIT: 5,
```

Comment:
Does this mean that all the melee spec that checks for crit hits somewhere should be updated to check for both crit hits and crit blocked?